# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-auxil/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-auxil/Cargo.toml b/third_party/rust/gfx-auxil/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/gfx-auxil/Cargo.toml
+++ /dev/null
@@ -1,36 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-edition = "2018"
-name = "gfx-auxil"
-version = "0.10.0"
-authors = ["The Gfx-rs Developers"]
-description = "Implementation details shared between gfx-rs backends"
-homepage = "https://github.com/gfx-rs/gfx"
-documentation = "https://docs.rs/gfx-auxil"
-keywords = ["graphics", "gamedev"]
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/gfx-rs/gfx"
-
-[lib]
-name = "gfx_auxil"
-[dependencies.fxhash]
-version = "0.2.1"
-
-[dependencies.hal]
-version = "0.9"
-package = "gfx-hal"
-
-[dependencies.spirv_cross]
-version = "0.23"
-optional = true